src/react-apps/applications/ux-editor/src/components/config/EditModalContent.tsx

```diff
@@ -229,6 +229,26 @@ class EditModalContentComponent extends React.Component<IEditModalContentProps,
           </Grid>
         );
       }
+      case 'Datepicker': {
```

Comment:
SonarCloud complains about this case is identical to the input-case (which is over). Just expand the input-case with 'Datepicker'.